src/api/common/index.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   dropsToXrp: utils.dropsToXrp,
   xrpToDrops: utils.xrpToDrops,
   toRippledAmount: utils.toRippledAmount,
-  generateWallet: utils.generateWallet,
+  generateAddress: utils.generateAddress,
```

Comment:
Isn't `ed25519` is supported now?
Looking into code I see you could call 
`generateAddress(type='ed25519')`